app/src/components/main/modal/darkModalLayout/modalHeader/modalHeader.scss

```diff
@@ -53,6 +53,8 @@
   cursor: pointer;
 
   svg {
+    width: 30px;
+    height: 30px;
```

Comment:
Is size inside of `.close-icon`  still needed?

Reply:
yes in safari this image doesn't show without size. As a different solution move it in the .svg file